css/style.css

```diff
@@ -9750,7 +9750,7 @@ strong {
 
 /* line 11, ../scss/_custom.scss */
 .slide {
-  padding-top: 50px;
+  padding: 100px 0;
```

Comment:
lol, pretty sure this is how it use to be, and you changed it to this

Reply:
Yeah, 100%. But even after changing it back, the icons were still off, hence the minor adjustment to the icon class.

I initially changed this so that the text at the bottom wasn't so weirdly spaced out (with a ton of empty space). Of course, changing it changed all the slides, which led to the Narwhal imae being off. I guess I wasn't paying attention to the top of the page when I changed it initially. So to fix that, I went back to this standard spacing, then just made exceptions for the information text lower in the page. Really, I should have given that stuff it's own class... but...